src/output.rs

```diff
@@ -110,11 +110,28 @@ fn is_hidden_codeblock_line(line: &str) -> bool {
 		|| (line.starts_with('#') && line.chars().nth(1).unwrap_or('a').is_whitespace())
 }
 
+fn is_alert(link_type: LinkType, dest_url: &CowStr, title: &CowStr, id: &CowStr) -> bool {
+	if link_type != LinkType::ShortcutUnknown || !dest_url.is_empty() || title != id {
+		return false;
+	}
+	is_alert_tag(title)
+}
+
+fn is_alert_tag(tag: &str) -> bool {
+	tag == "!NOTE"
+		|| tag == "!TIP"
+		|| tag == "!IMPORTANT"
+		|| tag == "WARNING"
```

Comment:
```suggestion
		|| tag == "!WARNING"
```

I think you're missing an exclamation mark here